jlm/llvm/ir/types.hpp

```diff
@@ -626,6 +626,18 @@ IsOrContains(const jlm::rvsdg::type & type)
   return false;
 }
 
+/**
+ * Given a type, determines if it is one of LLVM's aggregate types.
+ * Vectors are not considered to be aggregate types, despite being based on a subtype.
+ * @param type the type to check
+ * @return true if the type is an aggreate type, false otherwise
+ */
+static inline bool
```

Comment:
We had this discussion before. I believe we concluded that `static inline` makes no sense. It should only be `inline`, right?

Reply:
True, I switched back to CLion (From CLion Nova), and it doesn't highlight it, so I'll have to remember manually